DataFormats/SiStripDetId/interface/SiStripDetId.h

```diff
@@ -112,18 +112,18 @@ class SiStripDetId : public DetId {
 // ---------- inline methods ----------
```

Comment:
@civanch sorry, do you refer to this comment


The functions have the same number of returns as in the previous program version 
and the inline expression is never used.

I had  thought to change the <code>SiStripDetId::stereo()</code> function to

<code>
inline uint32_t SiStripDetId::stereo() const {
   return ( ((id_>>sterStartBit_ ) & sterMask_ ) == 1 ) ? 1:0;
}
</code>

but I am not sure whether the readability would suffer